src/components/common/form/FormDatePicker.tsx

```diff
@@ -0,0 +1,46 @@
+import { TextField } from '@mui/material'
+import { LocalizationProvider, DatePicker } from '@mui/x-date-pickers'
```

Comment:
I think they had an idea to monetize `@mui/x` components and this might become paid in the future
https://mui.com/x/introduction/
From what I am seeing from the documentation though it seems to be free!

Reply:
@dimitur2204 I guess if it becomes paid, then we'll stick to the free npm version or we will replace it with another fish in the sea of date picker libraries. :fishing_pole_and_fish:

Reply:
Thanks @dimitur2204 for the insight, I had no idea 😄 I agree with @kachar to leave it as it is for now and act accordingly if it becomes paid